htdocs/webhook/core/modules/webhook/mod_target_advanced.php

```diff
@@ -0,0 +1,146 @@
+<?php
```

Comment:
The number of different target we will use for webhook will be 1 to 5.
So there is no need to use a numbering module, to add a ref on a "setup" table.
Can you remove it and use the id as ref for entries into table llx_webhook_target ?